UnlimitedBacon.STL-Thumb version 0.4.0

SM-26 opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/29077)

SM-26 committed:
UnlimitedBacon.STL-Thumb version 0.4.0

Trenly commented on manifests/u/UnlimitedBacon/STL-Thumb/0.4.0/UnlimitedBacon.STL-Thumb.installer.yaml:
```diff
@@ -0,0 +1,13 @@
+PackageIdentifier: UnlimitedBacon.STL-Thumb
+PackageVersion: 0.4.0
+Installers:
+- Architecture: x86
```
I believe this installer works on x64 also
```suggestion
- Architecture: neutral
```

SM-26 replied:
The git page say x64
https://github.com/unlimitedbacon/stl-thumb#windows

SM-26 committed:
Update UnlimitedBacon.STL-Thumb.installer.yaml

update to be x64 based on this:
https://github.com/unlimitedbacon/stl-thumb#windows
